### ISSUE TITLE
Quest 8308 Brann Bronzebeard's Lost Letter shown as available when it is not

### ISSUE THREAD

**Issue by TorpedoCXC** [Questie/Questie#6503]
### Description

https://www.wowhead.com/cata/quest=8308/brann-bronzebeards-lost-letter

![WoWScrnShot_101324_223453](https://github.com/user-attachments/assets/ec7d1c61-0e0c-446c-abea-641742b74eae)

### Questie version

v10.6.0

### Game flavor

Cata

**Comment by Cabro** [comment]
It is available, just that it doesn't start from the NPC.

**Comment by TorpedoCXC** [comment]
Yes. This quest was started when I found the letter randomly dropped from a mob. It has to be handed in to this NPC then.
